TheJumpCloud/support
# fix: corrupted mac installation from jc admin console deployment script

## Issues
* [NB-10865](https://jumpcloud.atlassian.net/browse/NB-10865) - [JC AGENT] DA app installed from JC's admin console deployment script is being corrupted on macOS

## What does this solve?
DA app installed from JC's admin console deployment script is being corrupted on macOS

## Is there anything particularly tricky?
No

## How should this be tested?
**Prerequisites:**
1- macOS machine (preferably macOS Sonoma)

**Testing steps:**
1- Take the command from `Powershell/JumpCloud Commands Gallery/Mac Commands/Mac - Install JumpCloud Password Manager App.md`
2- Create a file on your machine and name it whatever you want but the extension should be `.sh`. e.g: `example.sh`
3- Paste the command you copied in step 1 in the `.sh` file you created in step 2
4- Open your terminal and go to the directory where the `.sh` file you created in step 2 can be found
5- Run `chmod 777 <scriptFileName>.sh` or `sudo chmod 777 <scriptFileName>.sh` depending on your situation
6- Run `./<scriptFileName>.sh`
7- After step 6 finishes execution, an app will show on your desktop. Double click it and see if the app opens successfully.

## Screenshots
N/A


[NB-10865]: https://jumpcloud.atlassian.net/browse/NB-10865?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
